torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -475,6 +477,14 @@ static bool is_int_list(PyObject* obj, int broadcast_size) {
   return broadcast_size > 0 && THPUtils_checkLong(obj);
 }
 
+static bool is_int_or_symbolic_or_concrete_int(PyObject* obj) {
+  if (THPUtils_checkLong(obj)) {
+      return true;
+  }
```

Comment:
TODO: test if it's the Python binding for SymbolicIntNode :)

Reply:
heh added a comment here